tests/installation/reconnect_s390.pm

```diff
@@ -40,6 +40,20 @@ sub run() {
         select_console('iucvconn');
     }
     else {
+        if (get_var('ENCRYPT')) {
+            my $password = $testapi::password;
+            my $svirt    = select_console('svirt');
+            my $name     = $svirt->name;
+            $svirt->suspend;
+            type_string "export pty=`virsh dumpxml $name | grep \"console type=\" | sed \"s/'/ /g\" | awk '{ print \$5 }'`\n";
+            type_string "echo \$pty\n";
+            $svirt->resume;
+
+            wait_serial("Please enter passphrase for disk.*", 100);
+            type_string "echo $password > \$pty\n";
+            wait_serial("Please enter passphrase for disk.*", 100);
+            type_string "echo $password > \$pty\n";
+        }
```

Comment:
shouldn't that better be moved into the encrypted password typing module? typing password for encrypt is not really something that we would expect in "reconnect_s390", is it?